root/static/scripts/edit/URLCleanup.js

```diff
@@ -1120,11 +1120,25 @@ const CLEANUPS: CleanupEntries = {
       },
     ],
     clean: function (url) {
+      // To reject /videoframe links without breaking the ?video_id parameter
+      if (/^https?:\/\/bandcamp.com\/videoframe/.test(url)) {
+        return url;
+      }
       url = url.replace(/^(?:https?:\/\/)?([^\/]+\.)?bandcamp\.com(?:\/([^?#]*))?.*$/, 'https://$1bandcamp.com/$2');
       url = url.replace(/^https:\/\/([^\/]+)\.bandcamp\.com\/(?:((?:album|track)\/[^\/]+))?.*$/, 'https://$1.bandcamp.com/$2');
       return url;
     },
     validate: function (url, id) {
+      if (/^https:\/\/(?:[^\/]+\.)?bandcamp\.com\/videoframe/.test(url)) {
+        return {
+          error: l(
+            `Please do not link to /videoframe links directly,
+             link to the appropriate track page instead.`,
+          ),
```

Comment:
I was trying to improve the formatting, and ended up rephrasing it in a generic way too:

```suggestion
          error: exp.l(
            `Please do not add “{blocked_url_pattern}” links,
             link to the appropriate “{wanted_url_pattern}” page instead.`,
            {
              blocked_url_pattern: (
                <span className="url-quote">{'/videoframe'}</span>
              ),
              wanted_url_pattern: (
                <span className="url-quote">{'/track'}</span>
              ),
            },
          ),
```

Reply:
This seems good - we might want to reuse it for some other strings later but we can start with this.